src/vim9class.c

```diff
@@ -2856,6 +2907,19 @@ object_free_nonref(int copyID)
     return did_free;
 }
 
+/*
+ * Output message which takes a variable name and the class that defines it.
+ * "cl" is that class where the name was found. Search "cl"'s hierarchy to
+ * find the defining class.
+ */
+    void
+emsg_var_cl_define(char *msg, char_u *name, size_t len, class_T *cl)
+{
+    ocmember_T	*m;
+    class_T	*cl_def = class_defining_member(cl, name, len, &m);
```

Comment:
The class_defining_member() function may return a NULL pointer? Do we need a check here cl_def == NULL?

Reply:
It is only null if the class/object does not have a member by the  name. Considering the first sentence of the comment

> Output message which takes a variable name and the class that defines it.

I don't see how it could be null.

Could have it return OK/FAIL (instead of void). Do nothing if NULL and return FAIL (if you're in a defensive mood). Which today would be ignored by all callers.

Reply:
One option would be to report E340, it is a better choice than SEGV.

Reply:
Using the E340 message sounds good to me.

Reply:
> Using the E340 message sounds good to me.

Done.